JM/CF/Scripts/3_Game/CommunityFramework/RPC/RPC_Context.c

```diff
@@ -0,0 +1,63 @@
+class CF_RPC_Context extends ScriptRPC
+{
+	protected string m_HandlerType;
+	protected string m_FunctionName;
+	protected bool m_Guaranteed;
+	protected Object m_Target;
+
```

Comment:
add a protected constructor here

Reply:
Adding a protected constructor results in `Method 'CF_RPC_Context' is private` compile error here: https://github.com/Jacob-Mango/DayZ-Community-Framework/blob/f9deb5e56a43e898fb1cb63ad1a43599b4bcd765/JM/CF/Scripts/3_Game/CommunityFramework/RPC/RPC_Context.c#L52